### PR TITLE
Format array entries uniformly

### PR DESCRIPTION
**Pull request by scottcwilson**

### CODE REVIEW

File: admin/customers.php [hunk 1]
```diff
@@ -212,7 +212,11 @@
                         'value' => $customers_telephone,
                         'type' => 'stringIgnoreNull'
                     ),
-                    array('fieldName' => 'customers_fax', 'value' => $customers_fax, 'type' => 'stringIgnoreNull'),
+                    array(
+                       'fieldName' => 'customers_fax',
+                       'value' => $customers_fax,
+                       'type' => 'stringIgnoreNull'
```
[the remark this answers is not in the document]
Comment (scottcwilson) [reply admin/customers.php:218]: The other entries didn't do it and I just wanted to fix this oddball alignment that was driving me crazy.

[the remark this answers is not in the document]
Comment (lat9) [reply admin/customers.php:218]: Building on what @mc12345678 said, why not take the opportunity to change the `array()` constructs to their more-current `[]` format?

[the remark this answers is not in the document]
Comment (scottcwilson) [reply admin/customers.php:218]: Really just wanted to fix the alignment.  Someone else is welcome to change `array()` to `[]` in a future PR.

**Comment by torvista** [comment]
This is a good example of you doing whatever you like/something pointless on a complete whim and no-one is really bothered that you have decided to invest your time in that. That's your choice. 

But, if **I** did this I'd get an earful of comments like:

> these commits seem superfluous (and create unnecessary work for people upgrading) if the functionality is the same.

> Scott (and the other core committers) have limited amounts of time to do the uncompensated work of maintaining Zen Cart, so we encourage people to focus on fixing bugs. Each PR costs me time and energy, and I only have so much of those things.

You can do what you like with your time, but practise what you preach and do not disparage the efforts of others however minor: you need all the help you can get.

You can consider **negatively** someone's minor PR as a waste of your time, 
or **positively** that it's an no-effort merge and be thankful someone is still bothering.

It's your choice as to what attitude you think is the most constructive.

**Comment by proseLA** [comment]
i am not a fan of this PR.

the impetus of this PR was that the format was driving scott crazy.  this, IMO, is not an impetus.

as previously stated, it does not fix a bug, nor is it a worthwild refactor IMO.

we have coding standards and the lines changed do not even conform to those coding standards.

i previously attempted to reformat this file, and [we got into a long conversation (never resolved) about coding standards and the need for them.](https://github.com/zencart/zencart/pull/5166)  i closed that PR (not merged) as we could not come to a consensus.  

as i recollect, there are numerous places in this file for refactoring, making it sweeter (loved that syntactic sugar reference i read here recently, just wish i could use it in a sentence).  but i refused to do it until we could come to a decision on standards.

with this PR not even conforming to the currently accepted standards, i would not merge it; and if it continues to drive scott crazy to the point of him being institutionalized, i would be more than willing to send him a card, provided someone provide me the address.

**Comment by scottcwilson** [comment]
I always think of Dr. Demento at times like this.
https://www.youtube.com/watch?v=3Fn36l_z3WY&ab_channel=LyricsforLife

I'm not sure how this file got messed up formatting.  But it did.  It would be easier to read if the formatting were consistent.  That doesn't mean I want to boil the ocean and take the risk of fixing everything that's objectionable in this file.

**Comment by scottcwilson** [comment]
Alternately, if one of you gentlemen would like to submit a PR that fixes these formatting problems and switches to short array syntax, I would be glad to merge it and close this.

**Comment by mc12345678** [comment]
> I'm not sure how this file got messed up formatting. But it did. It would be easier to read if the formatting were consistent. That doesn't mean I want to boil the ocean and take the risk of fixing everything that's objectionable in this file.

At least if it is all uniformly "wrong" it would be easier and more understandable about how to make it "right". My comparison tool ignores positional changes as do others, so once properly "lined", the shift left-right is just visual. Gotta start somewhere!

**Comment by scottcwilson** [comment]
Closing in preparation for 1.5.8a build.

**Pull request closed without merging**